external/vcm/vcm/calc/vertical_flux.py

```diff
@@ -0,0 +1,149 @@
+import xarray as xr
+import numpy as np
+
+
+def _center_to_interface(f: np.ndarray) -> np.ndarray:
+    """Interpolate vertically cell centered data to the interface
+    with linearly extrapolated inputs"""
+    f_low = 2 * f[..., 0] - f[..., 1]
+    f_high = 2 * f[..., -1] - f[..., -2]
+    pad = np.concatenate([f_low[..., np.newaxis], f, f_high[..., np.newaxis]], axis=-1)
+    return (pad[..., :-1] + pad[..., 1:]) / 2
+
+
+def _convergence(eddy: np.ndarray, delp: np.ndarray) -> np.ndarray:
+    padded = _center_to_interface(eddy)
+    # pad interfaces assuming eddy = 0 at edges
+    return -np.diff(padded, axis=-1) / delp
+
+
+def convergence_cell_center(
+    eddy: xr.DataArray, delp: xr.DataArray, dim: str = "p"
+) -> xr.DataArray:
+    """Compute vertical convergence of a cell-centered flux.
+
+    This flux is assumed to vanish at the vertical boundaries
```

Comment:
I know you just moved these functions here—but could you remove the incorrect comments on lines 16 and 25? The `convergence_cell_center` function does not assume zero flux at the vertical boundaries. It linearly extrapolates from the bottom/top two levels to get the boundary value. See `_center_to_interface` func.

Reply:
Yeah no problem.